.github/workflows/verify.yml

```diff
@@ -50,5 +50,5 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@5f1fec7010f6ae3b84ea4f7b2129beb8639b564f # v3
         with:
-          version: v1.52
+          version: v1.53.2
```

Comment:
keep just `v1.53` then will get all patches and then we dont need to update everytime :)

Reply:
Done